spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    email { 'test@test.com' }
```

Comment:
What if we use Fakers instead of a static string.
for example: 
```
email    { Faker::Internet.unique.email }
```

This would generate random data

Reply:
what would be the benefit of using this?